### DISCUSSION TITLE
Trying to round an image with border radius but turns to oval

### DISCUSSION THREAD

**Discussion by acamberos1** [tailwindlabs/tailwindcss#10422] [Help]
Whats the easiest way to round a largish image? its around 1700 x 1969 pixels, and when i try to change height it either gets too large or immediately shrinks

**Comment by wongjn** [comment]
If you want the image circular, you would need to make it square first and then it would be circular with `rounded-full`: https://play.tailwindcss.com/6W2Q2OmHQ0. Otherwise, if there is something else going on with your layout, please consider reproducing the undesired behavior in a [Tailwind Play](https://play.tailwindcss.com) and then we can look at helping you out.
